### PR TITLE
Configuration changes

### PR DESCRIPTION
**Pull request by simeg**
Please review each commit by itself, because each commit is self-contained (kinda). These are all just suggestions, let me know if you guys want to change anything.

Also, in one commit I accidentally reformatted the entire `package.json`. Sorry. If you want to I can revert it and fix it up but I'd rather not since it'll require some work.

### CODE REVIEW

File: book.json [hunk 1]
```diff
@@ -1,24 +1,18 @@
 {
-
-    "title": "Mac OS X Setup Guide ",
-
-    "links": {
```
Comment (sb2nov) [inline book.json]: I think the link.home might be needed as we don't host the site at a root url

File: package.json [hunk 2]
```diff
@@ -1,57 +1,42 @@
...
+    "Gitbook",
+    "Setup",
+    "Instruction"
+  ],
+  "author": "Sourabh Bajaj",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/sb2nov/mac-setup/issues"
+  },
+  "contributors": [
```
Comment (sb2nov) [inline package.json]: let's remove this as it is incorrect

File: book.json (hunk 1 above)
Comment (simeg) [reply book.json]: Ok! Removing this commit then.

**Comment by simeg** [comment]
@sb2nov All you comments are dealt with.

@Kyslik Interesting. I did some research comparing "OS X" versus "macOS" on how much people search for them on Google. Turns out they are _both_ being search for between 100k-1M each month (I used a tool in Google AdWords for this which meant creating an account, so I can't share the results with a link unfortunately). If we look on how they are "trending" (not sure the definition exactly) [we can see](https://trends.google.com/trends/explore?q=macos,os%20x) that "OS X" is trending slightly more.

Now why am I mentioning this? Because I want this awesome guide to have the maximum amount of exposure. I want people to have the best possible development environments for their machines. And in this case I think the numbers speak for themselves so I vote for keeping OS X until the term macOS become more used :)

What do you guys say?

**Comment by simeg** [comment]
Also, since the Gruntfile was removed in 31380e4c8188e81f2071f366d7839de56b50d82c I added a commit removing all Grunt related dependencies leaving us with just **2 dependencies**. Awesome!

**Comment by sb2nov** [comment]
LGTM merging

@Kyslik I think @simeg's assessment on this one seems great so we should stick to OSX for now and then upgrade in some time.

**Comment by Kyslik** [comment]
@sb2nov, @simeg I use **OS X** myself (only), but I've seen Apple use only **macOS** thats why I brought it up.

**Comment by sb2nov** [comment]
Yeah, I think they just renamed it in the last release. I remember it vaguely from watching the keynote.

**Comment by Kyslik** [comment]
In the <strike>guide</strike> book itself there are usages of macOS and OS X in separate places :rofl: 

see here (macOS): http://sourabhbajaj.com/mac-setup/Xcode/
see here (OS X): http://sourabhbajaj.com/mac-setup/SystemPreferences/

What I was trying to express along my comments is that we should unify the name no matter if OS X or macOS. We should also write about it in contribution guide.

**Comment by simeg** [comment]
I agree @Kyslik, we should stick to the same terms. What term we use is not important (well in this case I think it kinda is), but I also like consistency!

**Pull request merged**